src/AcreBtcNew.ts

```diff
@@ -401,6 +402,33 @@ export default class AcreBtcNew {
       };
     }
 
+  /**
+   * Signs a ERC4361 hex-formatted message with the private key at
+   * the provided derivation path according to the Bitcoin Signature format
+   * and returns v, r, s.
+   */
+  async signERC4361Message({ path, messageHex }: { path: string; messageHex: string }): Promise<{
+    v: number;
+    r: string;
+    s: string;
+  }> {
+    const pathElements: number[] = pathStringToArray(path);
+    const message = Buffer.from(messageHex, "hex");
+    const sig = await this.client.signERC4361Message(message, pathElements);
+    console.log("sig", sig);
```

Comment:
Console log

Reply:
removed this one and a couple others from the previous feature